framework/packages/abstract-interface/build.rs

```diff
@@ -43,8 +56,54 @@ fn main() {
     // .unwrap();
 
     // We also verify that the local artifacts dir exists
-    assert!(std::fs::metadata(artifacts_path).is_ok(), "You should create an artifacts dir in your crate to export the wasm files along with the cw-orch library");
+    let read_dir = fs::read_dir(artifacts_path).expect("You should create an artifacts dir in your crate to export the wasm files along with the cw-orch library");
+
+    // Path to OUT_DIR
+    let out_dir = env::var("OUT_DIR").unwrap();
+
+    let creator = if let Ok(creator) = env::var("ABSTRACT_CREATOR") {
+        let public_key = bip32::Mnemonic::new(&creator, Default::default())
+            .map(|phrase| {
+                let seed = phrase.to_seed("");
+                let derive_path: bip32::DerivationPath = "m/44'/118'/0'/0/0".parse().unwrap();
```

Comment:
Do we have to add `coin_type` or `derive_path` as another env? IMO it's an overkill until someone needs it